scripts/build-apis.sh

```diff
@@ -18,5 +18,5 @@ echo "Building common Kubernetes API objects"
 common_config_output_dir=$ROOT_DIR/config
 
 controller-gen paths=$ROOT_DIR/apis/... \
-    crd:trivialVersions=true object:headerFile=$TEMPLATES_DIR/boilerplate.txt \
```

Comment:
umm.. This parameter is not present anymore in controller-gen Version: v0.7.0

I used controller-gen: v0.7.0 to execute `scripts/build-apis.sh` . We need this version of controller-gen because it supports map of map inside crd generation.